src/course-outline/configure-modal/VisibilityTab.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Form } from '@edx/paragon';
+import { FormattedMessage, injectIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+
+const VisibilityTab = ({ isVisibleToStaffOnly, setIsVisibleToStaffOnly, showWarning }) => {
+  const handleChange = (e) => {
+    setIsVisibleToStaffOnly(e.target.checked);
+  };
+
+  return (
+    <>
+      <h3 className="mt-3"><FormattedMessage {...messages.sectionVisibility} /></h3>
+      <hr />
+      <Form.Checkbox checked={isVisibleToStaffOnly} onChange={handleChange}>
+        <FormattedMessage {...messages.hideFromLearners} />
+      </Form.Checkbox>
+      <hr />
```

Comment:
This separator would be better served in the block below, only when the warning is shown

Reply:
Definitely!